### ISSUE TITLE
No translation in discover tab (according to Movie Info Language setting)

### ISSUE THREAD

**Issue by RudyBzhBzh** [Radarr/Radarr#8939]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Discover tab only shows english information and does not respect Movie Info Language setting like in existing Movies or add new.
Similar as issue #5036 which was solved...

### Expected Behavior

Have informations in configured language.

In the following picture, green it's correct, red is not.
Top left is from "Add new" tab & bottom left is from "Discover" tab :
![Capture_decran_2023-08-03_220753](https://github.com/Radarr/Radarr/assets/77280357/62ea0fc3-76a8-4bf2-ba39-a00ee2c3a240)

Also notice that movie picture/poster is not according configured language in both places.
At least, TMDB API is talking about this : [https://developer.themoviedb.org/docs/image-languages](url)

### Steps To Reproduce

Nothing special except not having "english" as wanted "movie info language" in settings

### Environment

```markdown
- OS: Docker lscr.io/linuxserver/radarr:latest
- Radarr: 4.6.4.7568
- Docker Install: yes
- Using Reverse Proxy: yes
- Browser: FF
```

### What branch are you running?

Master

### Trace Logs? **Not Optional**

see picture

### Trace Logs have been provided as applicable. Reports will be closed if the required logs are not provided.

- [X] I have read and followed the steps in the wiki link above and provided the required trace logs - the logs contain `trace` - that are relevant and show this issue.

**Comment by RudyBzhBzh** [comment]
> Cannot reproduce, works as intended with multiple languages including French.
> 
> ![image](https://user-images.githubusercontent.com/707714/258606008-7296eca8-ce00-4918-870e-5b57086220ff.jpg)

Thanks for the feedback.
Ok… well, I’m on docker with last version… so what can it be in my side ?!
I will try to change to another language like Spanish or German to see if it’s better.
Could it be some sort of cache I can clear ?

also going to take debug/trace if something relevant could be find this way… 
Any other idea ?

Can it be somehow related to the source used for discover ? (Btw, I tested TMDB, Trakt and IMDB with same result…)

Listening for ideas please 🙏 ?

**Comment by RudyBzhBzh** [comment]
> I'm also using docker both develop and nightly but with no import list added.

OK, same for me. No issue **IF** I use "native radarr recommandations" :

![Capture d’écran 2023-08-06 151927](https://github.com/Radarr/Radarr/assets/77280357/f3026fb5-9c04-4b2d-9f19-45da6a66053f)

Everything is correctly in french (except for covers... but it's nowhere correct in the app) :
![Capture d’écran 2023-08-06 152736](https://github.com/Radarr/Radarr/assets/77280357/2557f5fc-0efc-46f8-ab54-880af393a736)

**BUT** it's not working if you configure import lists (uncheck native radarr recommandations and add a list).
Not working for every type of list I tested : TMDB Popular, Trackt Popular, IMDB Popular, StevenLu
![Capture d’écran 2023-08-06 153301](https://github.com/Radarr/Radarr/assets/77280357/79c96341-8541-4a38-8aaf-5d1fcb21c52e)

Shouldn't it be in configured language as well ?

**Comment by RudyBzhBzh** [comment]
> The covers aren't localized, and I'm certain of a duplicate issue of another French user complaining of the same thing.
> 
> @bakerboy448 ?

My main issue here  is about title and overview translation. But if the cover ma could be localised (as they are when you go to TMDB website), it could be great.

**Comment by RudyBzhBzh** [comment]
Thanks a lot ! 👍
Waiting for it to be in the master branch (I’m wondering to switch from master to dev or even to nightly and how to revert back… and don’t want to make a mistake 😱)

**Comment by RudyBzhBzh** [comment]
Hi @mynameisbogdan,
Just updated from 4.7.5.7809 to 5.0.3.8127 (both master branch) and my issue is still there.
The fix should be included to this new version, correct ?
I still only have english titles & synopsis on the discover page when using import list (same with IMDB Popular, TMDB Popular and Trakt linaspurinis)
I also tried deleting all cache without success.
Any idea please ?

**Comment by RudyBzhBzh** [comment]
> Oh man, it's been 2 months already?
> 
> Having `Movie Info Language` set to French works for me in Discovery.

Yes, 2 months… so sorry but I was waiting to be sure it was included in the master branch 🤷🏻‍♂️
I Don’t understand why it’s not working for me 😭
It works exactly as before : ok for radarr recommandations but not for import lists, proof that I correctly set it to French for « movie info settings ».

Don’t know what I’m doing wrong ?!

**Comment by Qstick** [comment]
This is a result of movieMetadata objects that are not tied to Movies not getting refreshed via the MovieRefreshService. There is a to-do in there to fix this at some point

**Comment by RudyBzhBzh** [comment]
> This is a result of movieMetadata objects that are not tied to Movies not getting refreshed via the MovieRefreshService. There is a to-do in there to fix this at some point
> 
> Since they don't get refreshed, they don't grab translations

Sorry, I don't clearly understand...
You mean it will work as soon as the list will be refreshed ?

At the moment, if I click on a movies already grabbed, it opens correctly in french (but it's not the same view)
When I click on a movie not already grabbed, it opens a view to grab it => This view is not in wanted info language.
Btw, before clicking on anything, the list displays english titles => EVEN for list I did not activated for a while (so probably freshly retrieved ; I it's not working for them also.

Well I'm lost :D 

Just tell me if I just have to wait a bit more (for some refreshes) or if a bug is still present and I have to wait for it to be addressed (If so, this issue should not be closed)

Thanks to you both.

**Comment by RudyBzhBzh** [comment]
> BTW are you using any import lists for Discover? Or you just want the results from TMDb?

It should be working for any kind of list, even custom ones, no ?
I’m not sure I understand what you mean.

**Comment by RudyBzhBzh** [comment]
> I mean if you have any enabled import lists without automatic add enabled, then [e78bc34](https://github.com/Radarr/Radarr/commit/e78bc345141f98632d65d0aa27772ccf45ef2ef9) should fix your issue.

I do not have any kind of automatic add, also I only display movies I do not already have in discovery. If I correctly understand, that is the reason why translate is not done (except for Radarr recommendations, which is not an import list).
Thanks a lot for the fix, hope it will work… but will have to wait for a while for me to test/confirm it, until it’s in the master branch… 
Please be comprehensive if I come back to you about this in a few weeks 😊

**Comment by RudyBzhBzh** [comment]
Just created a new docker instance, based on nightly.
And I'm sorry to tell you that it doesn't work for me.
